javascript/mcp-server/src/install/windsurf.ts

```diff
@@ -0,0 +1,52 @@
+import path from 'path'
+import os from 'os'
+import fs from 'fs'
+
+function getWindsurfConfigFilePath(): string {
+  // Ref: https://docs.codeium.com/windsurf/mcp
+  // NOTE: the official docs don't say where to put the file on Windows, so currently we don't support that.
```

Comment:
Should we log an error here if the platform is windows?

Reply:
Done. I wonder how hard it would be to put windows on the NUC we aren't using, would be nice to have a testing env.